DataBase.rb

```diff
@@ -0,0 +1,174 @@
+require 'active_record'
+require 'json'
+
+def db_init
+
+  ActiveRecord::Base.logger = Logger.new(File.open('database.log', 'w'))
+
+  ActiveRecord::Base.establish_connection(
+      :adapter => 'sqlite3',
+      :database => 'data.db'
+  )
+
+  ActiveRecord::Schema.define do
+
+    unless ActiveRecord::Base.connection.tables.include? 'pull_requests'
+      create_table :pull_requests do |t|
+        t.column :title, :string, :null => false
+        t.column :pr_id, :string, :null => false
+        t.column :author, :string, :null => false
+        t.column :merged, :boolean, :null => false
+        t.column :mergeable, :boolean, :null => true
+        t.column :mergeable_state, :string, :null => true
+        t.column :state, :string, :null => true
+        t.column :pr_commentors, :string, :null => true
+        t.column :committer, :string, :null => true
+        t.column :labels, :string, :null => true
+        t.column :created_at, :string, :null => true
+        t.column :updated_at, :string, :null => true
+        t.column :added_to_database, :string #:null => true
+
+      end
+    end
+
+    unless ActiveRecord::Base.connection.tables.include? 'users'
+      create_table :users do |table|
+        table.column :user_login, :string, unique: true
+        table.column :user_email, :string
+        table.column :git_email, :string
+        table.column :git_hub_id, :integer
+        table.column :notify_at, :string
+        table.column :daily_status, :boolean
+      end
+    end
+
+    unless ActiveRecord::Base.connection.tables.include? 'commentors'
+      create_table :commentors do |table|
+        table.column :pull_request_id, :integer #foreign key
+        table.column :user_id, :string
+      end
+    end
+
+  end
+end
+
+class User < ActiveRecord::Base
+  belongs_to :commentors
+end
+
+class PullRequest < ActiveRecord::Base
+  has_many :commentors
+end
+
+class Commentor < ActiveRecord::Base
+  belongs_to :pull_request
+  has_one :users
+end
+
+def add_new_pull_request i, pr_data
+  PullRequest.create(
+      :title => i.title,
+      :pr_id => i.number,
+      :author => i.user.login,
+      :merged => pr_data[:merged],
+      :mergeable => pr_data[:mergeable],
+      :mergeable_state => pr_data[:mergeable_state],
+      :state => pr_data[:state],
+      :pr_commentors => pr_data[:commentors].to_a.join(", "),
+      :committer => pr_data[:committer].to_a.join(", "),
+      :labels => pr_data[:label].to_a.join(", "),
+      :created_at => pr_data[:created_at],
+      :updated_at => pr_data[:updated_at],
+      :added_to_database => Time.new,
+
+  )
+
+  commentors_list = pr_data[:commentors].to_a
+  commentors_list.each do |c|
+    user = User.where(user_login: c)
+    Commentor.create(
+        :pull_request_id => i.number,
+        :user_id => user.first.id,
+    )
+  end
+end
+
+def check i, pr_data
```

Comment:
What doesn't this method check? Method name is not descriptive.

What does parameter 'i' do? Add more descriptive name.